PURL-TYPES.rst

```diff
@@ -586,6 +586,50 @@ swift
       pkg:swift/github.com/Alamofire/Alamofire@5.4.3
       pkg:swift/github.com/RxSwiftCommunity/RxFlow@2.12.4
 
+
+YOCTO Linux
+-----------
+``yocto`` for Yocto Linux recipes:
+
+The default repository is: ``git.yoctoproject.org``
+
+:namespace: The namespace is the name of the layer which provides the recipe.
+:name: The name is the name of the recipe.
+:version: The version is a commit or tag
+
+Qualifiers
+^^^^^^^^^^
+``repository``
```

Comment:
Other purl types use the qualifier `repository_url`, is there a specific reason to not use the same name here and have `repository` only?

Reply:
I was not sure because of the missing protocol. But maybe it's better to rename it and the protocol is mandatory.

Reply:
+1 for making it mandatory (or at least suggesting its use), explicit is better than implicit :-)